packages/mobile/src/store/drawers/selectors.ts

```diff
@@ -2,8 +2,10 @@ import type { AppState } from '../'
 
 import type { Drawer, DrawerData } from './slice'
 
-export const getData = <TDrawer extends keyof DrawerData>(state: AppState) =>
-  state.drawers.data as DrawerData[TDrawer]
+export const getData = <TDrawer extends keyof DrawerData>(
```

Comment:
nit: I think typically selectors do `state` as the first argument always?

Although I see below we wrap the selector in a new fn so 🤷 idk what's the best strategy here

Reply:
yeah was also a bit confused by the next selector below.